cvat/apps/webhooks/views.py

```diff
@@ -179,6 +188,7 @@ def redelivery(self, request, pk, delivery_id):
 
     @extend_schema(
         summary="Method send ping webhook",
```

Comment:
@kirill-sizov , could you please comment the use of empty request in this endpoint? I couldn't find tests.

Reply:
This request has no body because it's just a API method that we call to trigger `ping` [event](https://opencv.github.io/cvat/docs/administration/advanced/webhooks/#ping-webhook). Test is [here](https://github.com/opencv/cvat/blob/c7125a8ffbef2945a2256bc584f75353dd69ea1e/tests/python/rest_api/test_webhooks_sender.py#L688-L709)

Reply:
Is the request used at all?

Reply:
Ok, thanks